src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx

```diff
@@ -406,11 +406,12 @@ To edit environment variables, use these values:
 
 ### Browser agent [#browser-endpoints]
 
-To configure the browser agent to use a FedRAMP-compliant endpoint, you must use the [copy-paste method](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#copy-paste-app) method (other browser agent install methods are not supported) and edit the browser code’s script element tag so that the domain is `gov-bam.nr-data.net` for both `beacon` and `error_beacon`, like this:
+To configure the browser agent to use a FedRAMP-compliant endpoint, you must use the [copy-paste method](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#copy-paste-app) method (other browser agent install methods are not supported) and edit the browser code’s script element tag so that the domain is `gov-bam.nr-data.net` for both `beacon` and `errorBeacon`, like this:
```

Comment:
updated error_beacon to errorBeacon to match the prop name in the script.  But can revert if desired.

Reply:
Good catch!